client/sagas/err-saga.js

```diff
@@ -1,20 +1,14 @@
-//   () =>
-//     (store: Store) =>
-//     (next: (action: Action) => Object) =>
-//     errSaga(action: Action) => Object|Void
-export default () => ({ dispatch }) => next => {
-  return function errorSaga(action) {
-    const result = next(action);
-    if (!action.error) { return result; }
-
-    console.error(action.error);
-    return dispatch({
+export default function errorSaga(action$) {
+  return action$
+    .filter(({ error }) => !!error)
+    .map(({ error }) => error)
+    .doOnNext(error => console.error(error))
```

Comment:
:question: Can you also just use `.doOnNext(console.error)` or is that not valid?